account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_parser.py

```diff
@@ -351,9 +351,14 @@ def _convert_line_to_transactions(self, line):
             note = note.strip()
         if note:
             transaction["note"] = note
+        if notes:
+            transaction['note'] = notes
```

Comment:
This is wrong, as you're overwriting previous value. That should have been fixed by replacing
```
note = "{}\n{}".format(note, note.strip())
```
with
```
note = "{}\n{}".format(note, notes.strip())
```